practice_app/apps/user/templates/user-list.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  	<title>Home</title>
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+	<script src="//netdna.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
+	<link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
+	
+	<style type="text/css">
+		table, th, td {
+            border: 1px solid;
+        }
+        .center {
+            margin-left: auto;
+            margin-right: auto;
+            justify-content: center;
+        }
+        form { 
+            margin: 0 auto; 
+            width:250px;
+        }
+
+	</style>
+</head>
+<body>
+	<div class="wrapper">
+        <div style="background-color:#ADD8E6; !important" class="jumbotron text-center bg-primary">
+            <h1> User Portal </h1>
+            <p>
+				Welcome!
+            </p>
+        </div>
+	</div>
+	<div class="container">
+		<div class="container-fluid">
+            <div class="container">
+                <table class="center">
+                   <colgroup>
+						<col span="1" style="width: 10%;">
+						<col span="1" style="width: 30%;">
+						<col span="1" style="width: 30%;">
+						<col span="1" style="width: 30%;">
+                	</colgroup>
+                    <tr>
+						<th>ID</th>
```

Comment:
Similar to [user-detail.html](https://github.com/bounswe/bounswe2022group6/pull/108/files#diff-fa8b0310b07644261caf6e05c9344c0d451454214447a7b1d58b2d2d94be38ef), you can use some style for a better visual.